test/unit/org/apache/cassandra/schema/SchemaComparisonBetweenSSTablesAndCQLTablesTest.java

```diff
@@ -0,0 +1,364 @@
+
+    /**
+     * Table schema:
+     * CREATE TABLE test_5(
+     * col_txt text primary key,
+     * col_ascii ascii,
+     * col_bigint bigint,
+     * col_blob blob,
+     * col_bool boolean,
+     * col_date date,
+     * col_dec decimal,
+     * col_dbl double,
+     * col_float float,
+     * col_inet inet,
+     * col_small smallint,
+     * col_time time,
+     * col_timestamp timestamp,
+     * col_timeuuid timeuuid,
+     * col_tinyint tinyint,
+     * col_varchar varchar,
+     * col_varint varint
+     * );
+     */
+    private static final String TABLE_5_NAME = "test_5";
+
+    private static final Map<String, TableMetadata> ssTableMetadataForTableMap = new HashMap<>();
+
+    @BeforeClass
+    public static void init() throws Exception
+    {
+        DatabaseDescriptor.toolInitialization();
+        initializeSSTableMetadataForTableMap();
+    }
+
+    private static void initializeSSTableMetadataForTableMap() throws Exception
+    {
+        ssTableMetadataForTableMap.put(TABLE_1_NAME, Util.metadataFromSSTable(Descriptor.fromFilename("src/resources/org/apache/cassandra/schema/schema_validation_tests/test_1-7ab21491848211ee825e31532694b0d3/bb-2-bti-Data.db"), encodedKeyspaceName, TABLE_1_NAME));
+
+        ssTableMetadataForTableMap.put(TABLE_2_NAME, Util.metadataFromSSTable(Descriptor.fromFilename("src/resources/org/apache/cassandra/schema/schema_validation_tests/test_2-83ac0dd1848211ee825e31532694b0d3/bb-1-bti-Data.db"), encodedKeyspaceName, TABLE_2_NAME));
+
+        ssTableMetadataForTableMap.put(TABLE_3_NAME, Util.metadataFromSSTable(Descriptor.fromFilename("src/resources/org/apache/cassandra/schema/schema_validation_tests/test_3-88b6cb80848211ee825e31532694b0d3/bb-1-bti-Data.db"), encodedKeyspaceName, TABLE_3_NAME));
+
+        ssTableMetadataForTableMap.put(TABLE_4_NAME, Util.metadataFromSSTable(Descriptor.fromFilename("src/resources/org/apache/cassandra/schema/schema_validation_tests/test_4-8bf57bc0848211ee825e31532694b0d3/bb-1-bti-Data.db"), encodedKeyspaceName, TABLE_4_NAME));
+
+        ssTableMetadataForTableMap.put(TABLE_5_NAME, Util.metadataFromSSTable(Descriptor.fromFilename("src/resources/org/apache/cassandra/schema/schema_validation_tests/test_5-9db47b90848211ee825e31532694b0d3/bb-1-bti-Data.db"), encodedKeyspaceName, TABLE_5_NAME));
+    }
+
+    // *** GENERAL VALIDATION SUCCESS TESTS *** //
+
+    @Test
+    public void testIdenticalSchemaPassesValidation_SinglePartKeyNoClustering()
+    {
+        TableMetadata ssTableMetadata = ssTableMetadataForTableMap.get(TABLE_1_NAME);
+        TableMetadata cqlTableMetadata = TableMetadata.builder(encodedKeyspaceName, TABLE_1_NAME, TableId.fromUUID(UUID.randomUUID())).addPartitionKeyColumn("col_txt", UTF8Type.instance).addRegularColumn("col_int", Int32Type.instance).addRegularColumn("col_uuiid", UUIDType.instance).addRegularColumn("col_bool", BooleanType.instance).addRegularColumn("col_dec", DecimalType.instance).build();
```

Comment:
In the IDE these lines get pretty long and you are forced to scroll on the right to read the entire table definition. I would suggest to go to a new line after each `add*` method call. E.g.:
```suggestion
        TableMetadata cqlTableMetadata = TableMetadata.builder(encodedKeyspaceName, TABLE_1_NAME, TableId.fromUUID(UUID.randomUUID()))
                                                      .addPartitionKeyColumn("col_txt", UTF8Type.instance)
                                                      .addRegularColumn("col_int", Int32Type.instance)
                                                      .addRegularColumn("col_uuiid", UUIDType.instance)
                                                      .addRegularColumn("col_bool", BooleanType.instance)
                                                      .addRegularColumn("col_dec", DecimalType.instance)
                                                      .build();
```

And I would do that for all the tests.

Reply:
Done.